src/app/view/components/dot-navigation/dot-navigation.component.ts

```diff
@@ -12,22 +12,37 @@ import { DotNavigationService } from './dot-navigation.service';
 })
 export class DotNavigationComponent implements OnInit {
     menu: Observable<DotMenu[]>;
+    @Output() onMenuLinkClicked = new EventEmitter<Event>();
```

Comment:
Don't use "on" in the name of the events, check the angular styleguide: https://angular.io/guide/styleguide#dont-prefix-output-properties